test/image-blueprints-bootc/layer2-source/group1/rhel94-bootc-source.containerfile

```diff
@@ -35,3 +35,11 @@ RUN firewall-offline-cmd --zone=public --add-port=22/tcp && \
     firewall-offline-cmd --zone=public --add-port=6443/tcp && \
     firewall-offline-cmd --zone=public --add-port=30000-32767/tcp && \
     firewall-offline-cmd --zone=public --add-port=30000-32767/udp
+
+# Install systemd configuration drop-ins to fix potential permission problems
+# when upgrading from older ostree commits to bootc container layers
+RUN mkdir -p /etc/systemd/system/sshd.service.d && \
```

Comment:
Two things:

- It's preferred to use `usr/` over `etc/` for these things
- I think using the `COPY usr/ /usr/` pattern is cleaner (but it applies equally well to `COPY etc/ /etc/` as in other examples where config files need to be in `/etc`)

Reply:
We have templating mechanism in the build system, which does not support directory structure.
The build is run in an _output directory, so the .template file explicit copy is more straightforward because of that limitation.